client/src/commands/switch_impl_intf.ts

```diff
@@ -0,0 +1,50 @@
+import * as fs from "fs";
+import { LanguageClient, RequestType } from "vscode-languageclient/node";
+import { window } from "vscode";
+
+interface CreateInterfaceRequestParams {
+  uri: string;
+}
+
+let createInterfaceRequest = new RequestType<
```

Comment:
Maybe this one can be reused/imported from the `create_interface.ts` file?

Reply:
Good point. I've fixed it. https://github.com/rescript-lang/rescript-vscode/pull/404/commits/40366f106adc2951ae74a79731ec8c356d712269